package.json

```diff
@@ -15,12 +15,12 @@
     "lint": "./node_modules/.bin/eslint ./*.js app config lib server tests",
     "lint:hbs": "ember-template-lint .",
     "start": "ember serve",
-    "test": "ember test",
-    "prepare": "husky install"
+    "test": "ember test"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.18.6",
```

Comment:
The project won't build on my machine without this dependency. I'm not sure if some version drift dropped explicit inclusion of this (and everyone just has a stale node_modules on their machine), but I had to include this dependency.